### DISCUSSION TITLE
服务器功能

### DISCUSSION THREAD

**Discussion by lao2010321** [Meloong-Git/PCL#4164] [提问] [answered]
可不可以本地运行服务端，像forge之类的

**Comment by Ad-closeNN** [comment]
如果是想直接PCL启动服务器的话，那么
[抱歉，PCL 是 Minecraft 游戏启动器，不是 Minecraft 服务器启动器](https://github.com/Hex-Dragon/PCL2/discussions/3991#discussioncomment-9718993)

**Comment by WForst-Breeze** [comment, marked as answer]
#706
